library/openeuler

```diff
@@ -0,0 +1,39 @@
+Maintainers: openEuler CloudNative SIG <cloudnative@openeuler.org> (@openeuler-mirror),
+             Martin Grigorov <mgrigorov@openeuler.sh> (@martin-g)
+GitRepo: https://github.com/openeuler-mirror/docker-official-image.git
+
+Tags: 23.09
+Architectures: amd64, arm64v8
+GitFetch: refs/heads/23.09-x86_64
+GitCommit: d949f1d14dea7ef42305e41750ab3195c2b9dedc
+amd64-File: Dockerfile-23.09-x86_64
+arm64v8-GitFetch: refs/heads/23.09-aarch64
+arm64v8-GitCommit: 4753d7a10a37f5336693da9c4f588d8af19a2e43
+arm64v8-File: Dockerfile-23.09-aarch64
+
+Tags: 22.03-lts
+Architectures: amd64, arm64v8
+GitFetch: refs/heads/22.03-lts-x86_64
+GitCommit: 3fcc224614068ffc13a53a2301087af406652e69
+amd64-File: Dockerfile-22.03-lts-x86_64
+arm64v8-GitFetch: refs/heads/22.03-lts-aarch64
+arm64v8-GitCommit: fbdcee20b5e76ead420416d8dcb1632818fc026d
+arm64v8-File: Dockerfile-22.03-lts-aarch64
+
+Tags: 22.03-lts-sp1
+Architectures: amd64, arm64v8
+GitFetch: refs/heads/22.03-lts-sp1-x86_64
+GitCommit: 066bba8e7ed398ebbd3536811f17e8147369e3a9
+amd64-File: Dockerfile-22.03-lts-sp1-x86_64
+arm64v8-GitFetch: refs/heads/22.03-lts-sp1-aarch64
+arm64v8-GitCommit: be000f02309378563d4528bbbd306bcb53076086
+arm64v8-File: Dockerfile-22.03-lts-sp1-aarch64
+
+Tags: 22.03-lts-sp2
```

Comment:
Oh, it might be the _number_ of versions -- would you be willing to update this PR to be _just_ the latest release and we can deal with the backfill as a follow-up instead?

(I'm assuming "sp1" and "sp2" are updates to the 22.03 branch and not separate versions unto themselves?)

Reply:
I've commented out all tags but 22.03-lts-sp2. Let's see !

The SPs are updates but the provided RPM packages are split into groups and just the ones in the "main" group are ABI backward compatible. Many "non-essential" packages might have backward incompatibilities.

Reply:
The latest release for openEuler is `22.03-lts-sp3`